### DISCUSSION TITLE
Maximum length of transcription

### DISCUSSION THREAD

**Discussion by OmerIed** [openai/whisper#2077] [Q&A]
Does whisper has a maximum length for an output (for a single inference of an input of 30 seconds)?
If there is a maximum, what is it? is it defined by number of tokens/characters?

**Comment by ryanheise** [comment]
`model.dims.n_text_ctx` indicates 448 tokens for a 30 second window, but half of that is reserved by the prompt, so that leaves 224 tokens for output.
